cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -96,14 +96,16 @@
   private final Map<String, Map<String, String>> environments;
   private final RunId twillRunId;
   private final V1ObjectMeta resourceMeta;
+  private final Map<String, Float> kubeScalingFactors;
```

Comment:
How about adding "resource" to the name, e.g. kubeResourceScalingFactors

Reply:
Would it be simpler to keep a pointer to conf in KubeTwillRunnerService and KubeTwillPreparer?

Then you would need to add const string for the keys 
kubernetes.request.memory.multiplier and kubernetes.request.cpu.multiplier to 
io.cdap.cdap.common.conf.Constants

Set the default value in 
cdap-common/src/main/resources/cdap-default.xml